pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1644,8 +1644,7 @@ func buildListener(opts buildListenerOpts) *xdsapi.Listener {
 // This allows external https traffic, even when port the port (usually 443) is in use by another service.
 func appendListenerFallthroughRoute(l *xdsapi.Listener, opts *buildListenerOpts, node *model.Proxy) {
 	// If traffic policy is REGISTRY_ONLY, the traffic will already be blocked, so no action is needed.
-	if features.EnableFallthroughRoute() &&
-		opts.env.Mesh.OutboundTrafficPolicy.Mode == meshconfig.MeshConfig_OutboundTrafficPolicy_ALLOW_ANY {
+	if features.EnableFallthroughRoute() && isAllowAny(node) {
```

Comment:
since this has been around for a release, we can nuke this feature flag.. whjat do you think @howardjohn ?

Reply:
I agree we should clean up the flags (and was planning to do so soon), but specifically not this one. See https://github.com/istio/istio/issues/14242. Having this enabled can cause some problems, so I would like to keep it around until they are fixed